syntax = "proto3";
package Jde.Http.Proto.FromClient;
option optimize_for = LITE_RUNTIME;

message Connect{
	uint32 request_id = 1;
	uint32 session_id = 2;
}
